README.md

```diff
@@ -320,6 +246,13 @@ require'nvim-treesitter.configs'.setup {
 EOF
 ```
 
+# External Modules
+
+Other modules can be installed as plugins.
+
+- [refactor](https://github.com/nvim-treesitter/refactor) - Refactoring and definition modules
+- [playground](https://github.com/nvim-treesitter/playground) - Treesitter integrated playground
```

Comment:
You could add your documentation plugin?

Reply:
I think we can just link to the wiki now :)

Reply:
I'm gonna wait to add that one. It's still super raw and is going to change a bit once we get language injection.